src/main/resources/config.yml

```diff
@@ -10,6 +10,7 @@ include-essentials-warps-in-list: false
 use-bstats: true
 
 commands-on-warp-expire: []
+warp-name-format: "^.{1,15}$"
```

Comment:
I think this would benefit for a comment for the users.  While you and I understand what this means, most users will probably not understand what this does.
E.g something along the lines:
```yaml
# this specifies a regular expression for what's allowed in a warp name
# you can read about regular expressions, e.g in: https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html
# If you do not know what you're doing, it's probably best to leave this value alone.
```